### PR TITLE
test: add integration tests

### PR DESCRIPTION
**Pull request by songjiaxun**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind documentation

**What this PR does / why we need it**:
Add integration tests.
Modify documentations.
This PR depends on this fix: https://github.com/kubernetes-csi/csi-driver-nfs/pull/87

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

### CODE REVIEW

File: test/sanity/run-test.sh [hunk 1]
```diff
@@ -52,4 +54,4 @@ bin/nfsplugin --endpoint "$endpoint" --nodeid "$nodeid" -v=5 &
 
 echo 'Begin to run sanity test...'
 readonly CSI_SANITY_BIN='csi-test/cmd/csi-sanity/csi-sanity'
-"$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.testvolumeparameters="$(pwd)/test/sanity/params.yaml" --csi.endpoint="$endpoint" --ginkgo.skip="should not fail when requesting to create a volume with already existing name and same capacity|should fail when requesting to create a volume with already existing name and different capacity|should work|should fail when the requested volume does not exist"
```
Comment (andyzhangx) [inline test/sanity/run-test.sh:55]: what does `ginkgo.noColor` mean?

Comment (songjiaxun) [reply test/sanity/run-test.sh:55]: Removing flag `ginkgo.noColor` will give you the following "colorful" results in the console, which is nice to locate failures.
image

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Comment by andyzhangx** [comment]
should add the integration test pipeline in test-infra

**Pull request merged**